src/App.tsx

```diff
@@ -17,18 +37,30 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter onFilter={setSort} />
```

Comment:
Ensure that the `onFilter` prop in the `TodoFilter` component correctly updates the filter state. This is crucial for the filtering functionality to work as expected.